### PR TITLE
[FIX] VennDiagram Widget: Doesn't show all zeros in some input.

### LINKED ISSUE #1099: Venn Diagram only contains zeros
##### Steps to reproduce

Load iris.tab and connect File → Select Rows → Venn Diagram. And connect File → Venn Diagram.
Set condition in Select Rows: `sepal_length > 5`.
##### Expected bahavior

Venn Diagram contains correct number of instances in each slice.
##### Actual behavior

Venn Diagram shows 0 instances in all slices.
##### Additional info (worksheets, data, screenshots, ...)

[venn-zeros.ows.txt](https://github.com/biolab/orange3/files/167812/venn-zeros.ows.txt)

screenshot - 03102016 - 08 02 55 pm

### PR DESCRIPTION
**Pull request by mudit5bachhawat**
Some issue with trigger which resulted in showing all zeros.

Fixes #1099

### CODE REVIEW

File: Orange/widgets/visualize/owvenndiagram.py [hunk 1]
```diff
@@ -218,6 +218,8 @@ def itemsetAttr(self, key):
         attr_index = combo.currentIndex()
         if attr_index >= 0:
             return model[attr_index]
+        elif combo.count() is 0:
+            return -1;
```
Comment (ales-erjavec) [inline Orange/widgets/visualize/owvenndiagram.py]: Please use `==` operator to compare integers for equality. The fact that small integers in cpython are singletons is an implementation detail that should not be relied upon.

**Comment by mudit5bachhawat** [comment]
Thanks. Is everything else all right? Is is ready for merge now?

File: Orange/widgets/visualize/owvenndiagram.py (hunk 1 above)
Comment (mudit5bachhawat) [inline Orange/widgets/visualize/owvenndiagram.py]: Thanks. Changes Done. Is everything else all right? Is it ready for merge now?

File: Orange/widgets/visualize/owvenndiagram.py [hunk 2]
```diff
@@ -298,7 +299,9 @@ def _itemsForInput(self, key):
 
         def items_by_key(key, input):
             attr = self.itemsetAttr(key)
-            if attr is not None:
+            if attr is -1:
```
Comment (kernc) [inline Orange/widgets/visualize/owvenndiagram.py]: is here.

Comment (mudit5bachhawat) [inline Orange/widgets/visualize/owvenndiagram.py]: My Bad.

**Comment by zidarsk8** [comment]
@mudit5bachhawat Just a hint, but if you need to add something to the previous commit (which is better than having two commits with the same title) you can do `git commit --amend` and it will just add the changes to the current HEAD.

**Comment by mudit5bachhawat** [comment]
@zidarsk8 Thanks for your hint. I rebuilt my base by smashing second last commit into last commit. Will use  `--amend` from next time.

**Comment by kernc** [comment]
@mudit5bachhawat you can do it now already:
```bash
git reset HEAD^
git commit --amend Orange/widgets/visualize/owvenndiagram.py
git push --force
```
Please make sure the commit message is [descriptive](https://github.com/biolab/orange3/blob/master/CONTRIBUTING.md#commit-messages) by itself. Simply

> Issue#1099 Resolved

is _not_ ok. 

Thanks!

**Comment by mudit5bachhawat** [comment]
@kernc, Is it all right now?

**Comment by mudit5bachhawat** [comment]
@kernc Is it okay now?

**Comment by ajdapretnar** [comment]
@mudit5bachhawat Venn now works fine with two Select Rows widgets, but it does not output data subset anymore. It might be an issue with your branch.....

**Comment by ajdapretnar** [comment]
@mudit5bachhawat Have you checked the issue at hand?

**Comment by mudit5bachhawat** [comment]
Need some time.

**Comment by ajdapretnar** [comment]
I am closing this PR. If you decide to work on this issue at any later time, you can reopen it.

**Pull request closed without merging**